packages/components/src/components/primitives/CircularProgress/CircularProgress.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { renderWithProviders } from 'tests/renderWithProviders';
+import { CircularProgress } from '.';
+
+describe('CircularProgress', () => {
+  test('should render', () => {
+    renderWithProviders(<CircularProgress value={10} />);
```

Comment:
Should we expect an assertion? @Angele91

Reply:
fixed: added an assertion for the render test